frontend/src/components/messstelle/charts/MessstelleDiagramme.vue

```diff
@@ -113,7 +115,6 @@
 
         <!-- Speed Dial alles außer Listenausgabe-->
         <speed-dial
-            v-show="showSpeedial"
```

Comment:
Das bitte noch drinnen lassen, damit der Button nicht in der Zeitreihe auftaucht

Reply:
hab die zeitreihe vergessen, ist wieder drinnen